torch/csrc/jit/passes/mkldnn_rewrite.cpp

```diff
@@ -125,48 +170,157 @@ void insertMkldnnPrePackedOps(script::Module& module) {
   }
 }
 
-void FuseReluWithPackedOps(std::shared_ptr<Graph>& graph) {
+template <typename T>
+void RewriteEltwiseGraph(
+    std::shared_ptr<Graph>& graph,
+    const std::map<std::string, T>& fusion_rewrite_map,
+    std::string prepack_op_name,
+    std::string run_op_name,
+    std::string op_context_name,
+    std::string graph_input,
+    std::string prepack_input) {
   auto conv_op_rstring = at::jit::CodeTemplate(R"(
-    graph(%input, %weight, %bias, %stride:int[], %padding:int[],
-          %dilation:int[], %groups:int, %input_size:int[], %dummy_attr:str):
-        %packed_weight_bias = mkldnn_prepacked::conv2d_prepack(
-            %weight, %bias, %stride, %padding, %dilation, %groups,
-            %input_size, %dummy_attr)
-        %conv2d_res = mkldnn_prepacked::conv2d_run(%input, %packed_weight_bias)
-        %res = aten::${op}(%conv2d_res)
+    graph(${graph_input}
+          %input_size:int[], %attr_placeholder:str, %scalars_placeholder: Scalar?[], %algorithm_placeholder: str?${op_input_str}):
+        %packed_weight_bias = ${prepack_op_name}(
+            ${prepack_input}
+            %input_size, %attr_placeholder, %scalars_placeholder, %algorithm_placeholder)
+        %conv2d_res = ${run_op_name}(%input, %packed_weight_bias)
+        %res = aten::${op}(%conv2d_res${op_input_str})
         return (%res))");
 
   auto conv_op_fused_rstring = at::jit::CodeTemplate(R"(
-    graph(%input, %weight, %bias, %stride:int[], %padding:int[],
-          %dilation:int[], %groups:int, %input_size:int[], %dummy_attr:str):
+    graph(${graph_input}
+          %input_size:int[], %attr_placeholder:str, %scalars_placeholder: Scalar?[], %algorithm_placeholder: str?${op_input_str}):
         %attr: str = prim::Constant[value="${op_attr}"]()
-        %packed_weight_bias : __torch__.torch.classes.mkldnn.ConvOpContext = mkldnn_prepacked::conv2d_prepack(
-            %weight, %bias, %stride, %padding, %dilation, %groups,
-            %input_size, %attr)
-        %res = mkldnn_prepacked::conv2d_run(%input, %packed_weight_bias)
+        ${construct_operand_list}
+        %packed_weight_bias : __torch__.torch.classes.${op_context_name} =  ${prepack_op_name}(
+            ${prepack_input}
+            %input_size, %attr, %scalars, %algorithm)
+        %res = ${run_op_name}(%input, %packed_weight_bias)
         return (%res))");
 
-  for (auto const& it : mkldnn::fusion_rewrite_map) {
+  for (auto const& it : fusion_rewrite_map) {
```

Comment:
What's the point of having "none" entry in the rewrite map if we're simply skipping it?

Reply:
the "none" entry path is not needed, I removed it.